ckanext/spatial/harvesters/base.py

```diff
@@ -285,6 +285,26 @@ def _extract_first_license_url(licences):
             if license_url_extracted:
                 extras['licence_url'] = license_url_extracted
 
+
+        # Metadata license ID check for package
+        use_constraints = iso_values.get('use-constraints')
+        if use_constraints:
+
+            license_list = logic.action.get.license_list({'model': model, 'session': model.Session, 'user': 'harvest'}, {})
```

Comment:
Can you use `p.toolkit.get_action('license_list')(context, {})` instead of calling the function directly? This will ensure that people can override it from other extensions.

Also please use ` self._get_user_name()` as the value of `user` on the context, not `harvest`